source/libraries/HelperFunctions.ts

```diff
@@ -80,9 +78,11 @@ async function sleep(milliseconds: number): Promise<object> {
 }
 
 export async function waitForTransactionToBeSealed(ethjsQuery: EthjsQuery, transactionHash: string): Promise<void> {
+    let pollingInterval = 10;
```

Comment:
Recommend testing this vs live nodes (Parity and geth) to make sure this doesn't overload their RPC servers.